src/android/java/com/pspdfkit/cordova/action/document/ShowDocumentAction.java

```diff
@@ -95,39 +95,87 @@ private PdfActivityConfiguration parseOptionsToConfiguration(@NonNull final JSON
-        } else if ("disableOutline".equals(option) && ((Boolean) value)) {
-          builder.disableOutline();
-        } else if ("disablePrinting".equals(option) && ((Boolean) value)) {
-          builder.disablePrinting();
-        } else if ("disableSearch".equals(option) && ((Boolean) value)) {
-          builder.disableSearch();
+          if ((Boolean) value) {
+            builder.disableCopyPaste();
+          } else {
+            builder.enableCopyPaste();
+          }
+        } else if ("disableDocumentEditor".equals(option)) {
+          if (((Boolean) value)) {
+            builder.disableDocumentEditor();
+          } else {
+            builder.enableDocumentEditor();
+          }
+        } else if ("disableOutline".equals(option)) {
+          if (((Boolean) value)) {
+            builder.disableOutline();
+          } else {
+            builder.enableOutline();
+          }
+        } else if ("disablePrinting".equals(option)) {
+          if (((Boolean) value)) {
+            builder.disablePrinting();
+          } else {
+            builder.enablePrinting();
+          }
+        } else if ("disableSearch".equals(option)) {
+          if (((Boolean) value)) {
+            builder.disableSearch();
+          } else {
+            builder.enableSearch();
+          }
         } else if ("shareFeatures".equals(option)) {
           builder.setEnabledShareFeatures(parseShareFeatures((JSONArray) value));
         } else if ("disableUndoRedo".equals(option)) {
           builder.undoEnabled(!(Boolean) value);
-        } else if ("hidePageLabels".equals(option) && ((Boolean) value)) {
-          builder.hidePageLabels();
-        } else if ("hidePageNumberOverlay".equals(option) && ((Boolean) value)) {
-          builder.hidePageNumberOverlay();
-        } else if ("hideSettingsMenu".equals(option) && ((Boolean) value)) {
-          builder.hideSettingsMenu();
+        } else if ("hidePageLabels".equals(option)) {
+          if (((Boolean) value)) {
+            builder.hidePageLabels();
+          } else {
+            builder.showPageLabels();
+          }
+        } else if ("hidePageNumberOverlay".equals(option)) {
+          if (((Boolean) value)) {
+            builder.hidePageNumberOverlay();
+          } else {
+            builder.showPageNumberOverlay();
+          }
+        } else if ("hideSettingsMenu".equals(option)) {
+          if (((Boolean) value)) {
+            builder.hideSettingsMenu();
+          } else {
+            builder.showSettingsMenu();
+          }
         } else if ("thumbnailBarMode".equals(option)) {
-          builder.setThumbnailBarMode(ThumbnailBarMode.valueOf((String) value));
-        } else if ("hideThumbnailGrid".equals(option) && ((Boolean) value)) {
-          builder.hideThumbnailGrid();
+          final CordovaThumbnailBarMode thumbnailBarMode = CordovaThumbnailBarMode.valueOf((String) value);
+          builder.setThumbnailBarMode(thumbnailBarMode.androidThumbnailBarMode);
```

Comment:
While merging the iOS and Android plugins into this repo, we changed the public API name for thumbnail bar options, leading to an error here. I added an adapter enum for mapping between JS and Android names.